packages/assistant/pyproject.toml

```diff
@@ -10,7 +10,7 @@ homepage = "https://rpaframework.org/"
 documentation = "https://rpaframework.org/"
 repository = "https://github.com/robocorp/rpaframework"
 
-keywords = ["robotframework", "rpa", "automation", "dialogs"]
+keywords = ["robotframework", "rpa", "automation", "assistant"]
```

Comment:
@xylix Let me know if this change is wrong :D.

Reply:
This is basically so that it will show up in: https://pypi.org/search/?q=dialogs or such searches. So I would actually keep the dialogs and add assistant on top of it